### PR TITLE
strip session-token in ceFormat

### LINKED ISSUE #360: [FEATURE] strip session-token in ceFormat
**What feature you'd like to add:**

**Why is this feature needed:**

### PR DESCRIPTION
**Pull request by Hexilee**
close #360

### CODE REVIEW

File: pkg/config/setting.go [hunk 1]
```diff
@@ -296,6 +297,53 @@ func ParseSetting(secrets, volCtx map[string]string, options []string, usePod bo
 	return &jfsSetting, nil
 }
 
+func (s JfsSetting) parseFormatOptions() ([][]string, error) {
+	options := strings.Split(s.FormatOptions, ",")
+	parsedOptions := make([][]string, 0)
+	for _, option := range options {
+		pair := strings.Split(strings.TrimSpace(option), "=")
```
Comment (zwwhdls) [inline pkg/config/setting.go:304]: Some options are not "key=value" form, do not have value.

**Review by zwwhdls — CHANGES_REQUESTED** [review]

File: pkg/config/setting.go [hunk 2]
```diff
@@ -296,6 +297,53 @@ func ParseSetting(secrets, volCtx map[string]string, options []string, usePod bo
 	return &jfsSetting, nil
 }
 
+func (s JfsSetting) parseFormatOptions() ([][]string, error) {
```
Comment (zwwhdls) [inline pkg/config/setting.go]: Add unit test please.

File: pkg/config/setting.go (hunk 1 above)
Comment (Hexilee) [reply pkg/config/setting.go:304]: Single-key options are supported.

File: pkg/config/setting.go (hunk 2 above)
Comment (Hexilee) [reply pkg/config/setting.go]: Added

**Review by zwwhdls — APPROVED** [review]

**Pull request merged**